hm_pyhelper/diagnostics/diagnostics_report.py

```diff
@@ -53,6 +54,30 @@ def append_error(self, key):
     def get_errors(self):
         return self[DIAGNOSTICS_ERRORS_KEY]
 
+    def assert_diagnostics_present(self, required_keys: set) -> bool:
+        """
+        Confirm that all keys specified are present in self.miner.payload.
```

Comment:
"self.miner.payload." - Perhaps this was left in doc string accidentally?

Reply:
Fixed: https://github.com/NebraLtd/hm-pyhelper/pull/93/commits/d1234c69734341c03efc7d864927debaa2ed409e